src/main/java/io/dropwizard/kafka/managed/KafkaAdminClientManager.java

```diff
@@ -12,15 +15,20 @@
 
     private final AdminClient adminClient;
     private final String name;
+    private final Collection<NewTopic> topics;
 
-    public KafkaAdminClientManager(final AdminClient adminClient, final String name) {
+    public KafkaAdminClientManager(final AdminClient adminClient, final String name, Collection<NewTopic> topics) {
         this.adminClient = requireNonNull(adminClient);
         this.name = requireNonNull(name);
+        this.topics = topics;
     }
 
     @Override
     public void start() throws Exception {
-        // do nothing
+        if (this.topics != null && !this.topics.isEmpty()) {
+            // TODO: Check if topics already exist
+            this.adminClient.createTopics(this.topics);
```

Comment:
Out of curiosity, what is the behavior of the client if `createTopics` is created every time the app starts up? Will it be a no-op if the topics already exist? Otherwise, the check might be pretty important.